Sample-WPF/FaceAPI-WPF-Samples.csproj

```diff
@@ -57,7 +57,7 @@
       <Private>True</Private>
     </Reference>
     <Reference Include="Microsoft.ProjectOxford.Face, Version=1.2.5.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
```

Comment:
I think there are many other places you need to change the version number. Please make sure all covered.